.npmignore

```diff
@@ -0,0 +1 @@
+packages/java-parser/resources/Unicode/*
```

Comment:
I think you want the .npmignore file right next to the relevant package.json
not at the root of the repository.